brozzler/browser.py

```diff
@@ -427,6 +427,7 @@ def browse_page(
             self.websock_thread.on_request = on_request
         if on_response:
             self.websock_thread.on_response = on_response
+        brozzler.thread_accept_exceptions()
```

Comment:
Would definitely split the implementation and use of this new feature separate commits in the future.